docs/source/changelog/bugfix/pipelined-startup-err.rst

```diff
@@ -0,0 +1,4 @@
+[Bugfix] Gracefully handle startup timeout
+==========================================
+
+* In the pipelined executor, increase default timeout and emit a more user-friendly error message in case of hitting the timeout (:pr:`1342`).
```

Comment:
I am not sure if we have a line length limit in the docs ? (this is 143 wide)

Reply:
Ah, right, we do try to keep lines short-ish, to have better diffs. I'll re-wrap.